### PR TITLE
Deprecate verbs parcelport

### PR DESCRIPTION
**Pull request by msimberg**
Flyby: fixes verbs and other parcelport related options.

### CODE REVIEW

File: cmake/toolchains/BGION-gcc.cmake [hunk 1]
```diff
@@ -41,8 +41,8 @@ set(HPX_PLATFORM "native")
 # Disable generic coroutines (and use posix version)
 set(HPX_WITH_GENERIC_CONTEXT_COROUTINES OFF CACHE BOOL "disable generic coroutines")
 
-# BGAS nodes support ibverbs
-set(HPX_WITH_PARCELPORT_IBVERBS ON CACHE BOOL "")
+# BGAS nodes support ibverbs, but it is deprecated
+set(HPX_WITH_PARCELPORT_VERBS OFF CACHE BOOL "")
```
Comment (hkaiser) [inline cmake/toolchains/BGION-gcc.cmake]: Should we enable the MPI parcelport by default instead?

File: cmake/toolchains/XeonPhi.cmake [hunk 2]
```diff
@@ -37,7 +37,7 @@ set(CMAKE_CROSSCOMPILING ON)
 set(HPX_PLATFORM "XeonPhi")
 
 # Always disable the ibverbs parcelport as it is non-functional on the BGQ.
-set(HPX_WITH_PARCELPORT_IBVERBS OFF CACHE BOOL "Enable the ibverbs based parcelport. This is currently an experimental feature")
+set(HPX_WITH_PARCELPORT_VERBS OFF CACHE BOOL "Enable the ibverbs based parcelport. This is currently an experimental feature")
```
Comment (hkaiser) [inline cmake/toolchains/XeonPhi.cmake]: Same here, should we enable MPI instead?

**Comment by biddisco** [comment]
just a comment. at some point we will remove the parcelport completely, however, it would be nice if we created an `hpx/obsolete` folder/module that contained useful, but no longer functional or tested code so that if we ever need it again, it was somewhat easier to find than going through old branches etc. (comments?)

**Comment by sithhell** [comment]
Why not just use tags? Or create a wiki page pointing to the commit before it got removed.

**Comment by biddisco** [comment]
compromise...?
in `hpx/obsolete/obsolete.txt`
Parcelport Verbs was removed in git SHA  f47b3814bf5db1ec483611397708f40f4cb57db3
(or maybe - the last commit before it was removed is .....)

**Pull request merged**